pkg/scheduler/framework/plugins/nodevolumelimits/csi.go

```diff
@@ -180,6 +180,14 @@ func (pl *CSILimits) filterAttachableVolumes(
 		pvc, err := pl.pvcLister.PersistentVolumeClaims(pod.Namespace).Get(pvcName)
 
 		if err != nil {
+			if newPod {
+				// The PVC is required to proceed with
+				// scheduling of a new pod because it cannot
+				// run without it. Bail out immediately.
+				return fmt.Errorf("look up PVC %s/%s: %v", pod.Namespace, pvcName, err)
```

Comment:
should we provide a little more context to the error message, like "failed to ..."?

Reply:
I tend to write error messages as "<operation>: <error>". Then if it gets wrapped higher up, we don't accumulate those "failed to " comments. I've seen errors which were something like:
```
ERROR: failed to <do abc>: failed to <do xyz>: could not <do fgh>: original error here
```

That always seemed a bit redundant to me. But let me add it here.... done.

Reply:
We tend to say "looking up PVC ...." because it is clear this is an error from the context, so adding "failed to" is redundant.

Reply:
I like "looking up PVC" best, so I've switched to that.

@msau42 okay? I still need LGTM for this PR.